utm.yaml

```diff
@@ -1317,6 +1271,213 @@ components:
         status:
           $ref: '#/components/schemas/UssAvailabilityStatus'
 
+    RequestResponse:
+      type: object
+      description: Details of a request/response data exchange.
+      required:
+      - url
+      - method
+      - request_time
+      - request_body
+      - response_time
```

Comment:
Not every request receives a response, so this field should be optional and omitted if a response was not received